app/case-studies/[details]/page.tsx

```diff
@@ -1,5 +1,39 @@
-const CaseStudy = () => {
-  return <div>Case Study</div>;
+"use client";
+
+import React from "react";
+
+import { featuredProjectsList } from "@/constants";
+import CaseStudyHeader from "@/app/components/case-studies-components/CaseStudyHeader";
+import CaseStudyTechStack from "@/app/components/case-studies-components/CaseStudyTechStack";
+import CaseStudyProblemStatement from "@/app/components/case-studies-components/CaseStudyProblemStatement";
+import CaseStudyChallengesAndLearnings from "@/app/components/case-studies-components/CaseStudyChallengesAndLearnings";
+import CaseStudiesOtherCaseStudies from "@/app/components/case-studies-components/CaseStudiesOtherCaseStudies";
+import GetInTouchCard from "@/app/components/homepage-components/GetInTouchCard";
```

Comment:
Imports can't be grouped if they're from different modules. However, if you find that you're importing multiple components from the same directory, it might be a good idea to create an  index.ts file in that directory that exports all these components. This way, you can import them all at once from that directory. Then, you can simplify your imports like so:

```
import {
  CaseStudyHeader,
  CaseStudyTechStack,
  CaseStudyProblemStatement,
  CaseStudyChallengesAndLearnings,
  CaseStudiesOtherCaseStudies,
} from "@/app/components/case-studies-components";

```
Let me know if you have any questions about this. Happy to help with this or anything!

Reply:
I've put these into an index.ts now. Always good to do some decluttering